tests/api.c

```diff
@@ -2698,6 +2698,12 @@ static void test_wolfSSL_ERR_peek_last_error_line(void)
 #endif
 
     printf(resultFmt, passed);
+
+    printf("\nTesting error print out\n");
+    ERR_print_errors_fp(stdout);
+    printf("Done testing print out\n\n");
+    fflush(stdout);
+    wolfSSL_Cleanup();
```

Comment:
The library can't be used again after Cleanup() call, is this correct and the correct spot?